src/meta-srv/src/service/admin/node_lease.rs

```diff
@@ -41,7 +41,8 @@ impl HttpHandler for NodeLeaseHandler {
             .into_iter()
             .map(|(k, v)| HumanLease {
                 name: k,
-                human_time: common_time::DateTime::new(v.timestamp_millis).to_string(),
+                human_time: common_time::Timestamp::new_microsecond(v.timestamp_millis)
```

Comment:
We have to use millisecond timestamp here as `v` is millisecond.

Reply:
Done. Thank you!

Reply:
@evenyag PTAL